test/UnitTest.cs

```diff
@@ -1769,16 +1768,16 @@ public void TestSort()
             Assert.IsTrue(TU.error_.Contains("appear"));
 
             sql = "select(4-a3)/2,(4-a3)/2*2 + 1 + min(a1), avg(a4)+count(a1), max(a1) + sum(a1 + a2) * 2 from a group by 1 order by 1";
-            result = ExecuteSQL(sql, out string phyplan);
-            var answer = @"PhysicOrder   (actual rows=2)
-                            Output: {4-a.a3/2}[0],{4-a.a3/2*2+1+min(a.a1)}[1],{avg(a.a4)+count(a.a1)}[2],{max(a.a1)+sum(a.a1+a.a2)*2}[3]
-                            Order by: {4-a.a3/2}[0]
-                            -> PhysicHashAgg   (actual rows=2)
-                                Output: {4-a.a3/2}[0],{4-a.a3/2}[0]*2+1+{min(a.a1)}[1],{avg(a.a4)}[2]+{count(a.a1)}[3],{max(a.a1)}[4]+{sum(a.a1+a.a2)}[5]*2
-                                Aggregates: min(a.a1[1]), avg(a.a4[2]), count(a.a1[1]), max(a.a1[1]), sum(a.a1[1]+a.a2[4])
-                                Group by: {4-a.a3/2}[0]
-                                -> PhysicScanTable a  (actual rows=3)
-                                    Output: 4-a.a3[2]/2,a.a1[0],a.a4[3],a.a1[0]+a.a2[1],a.a2[1],a.a3[2]";
+            result = ExecuteSQL(sql, out string phyplan);           
+            var answer = @"PhysicOrder  (actual rows=2)
+                            Output: {(4-a.a3)/2}[0],{(((4-a.a3)/2*2+1)+min(a.a1))}[1],{(avg(a.a4)+count(a.a1))}[2],{(max(a.a1)+sum((a.a1+a.a2))*2)}[3]
```

Comment:
((( -- too many but I guess we have to stay with it for now better than wrong impression of bad maths.